airbyte-config/config-models/src/main/java/io/airbyte/config/Configs.java

```diff
@@ -241,12 +245,23 @@ public interface Configs {
    */
   int getAirbyteApiPort();
 
+  /**
+   * Define the header name used to authenticate with the Airbyte API
```

Comment:
From parent PR: 
![image](https://user-images.githubusercontent.com/7798112/186285450-4764a126-a3ea-42bf-a22f-c06ae7eab9cd.png)

Reply:
Ah yes this is only used by the worker, I should probably just move these to the `// Worker` section of this file to make that clear.

Reply:
@cgardens forgot to tag you in the top of the new thread so tagging you now so you see my response

Reply:
sgtm